bin/MadGraph5_aMCatNLO/cards/production/13TeV/higgs/ggh01_M125_Toa01a01_M20_Tomumubb/ggh01_M125_Toa01a01_M20_Tomumubb_run_card.dat

```diff
@@ -89,6 +89,17 @@
 # Apply pt/E/eta/dr/mij cuts on decay products or not
 # (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
 #**********************************************************
+#    
+#*************************************************************
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
```

Comment:
I think for this particular mass point it was removed by mistake in the latest commits. It has been existing for all the rest (15,25,..,60) and now added also for 40 GeV